homeassistant/components/binary_sensor/lutron.py

```diff
@@ -0,0 +1,84 @@
+"""
+Binary sensor for Lutron RadioRA2 keypad buttons.
+
+There are two classes of button: raise/lower buttons that have discrete
+press/release events (e.g., dimmer buttons) and single-action buttons
+that only have press events.
+
+A sensor for a raise/lower button is "on" when the button is pressed and
+"off" when it's released.
+
+A sensor for a single-action button goes from "off" to "on" and then
+immediately back to "off" when the button is clicked, that is, it
+appears to be pressed and then released instantaneously.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/lutron/
+"""
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.components.lutron import LUTRON_DEVICES
+
+DEPENDENCIES = ['lutron']
+REQUIREMENTS = ['pylutron==0.2.0']
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the Lutron RadioRA2 binary sensor platform."""
+    if LUTRON_DEVICES not in hass.data:
```

Comment:
This is not possible, because it means the component was not set up correctly, which would cause the platform to be never loaded.